src/bin/vip-logs.js

```diff
@@ -86,6 +109,7 @@ command( {
 } )
 	.option( 'type', 'The type of logs to be returned: "app" or "batch"', 'app' )
 	.option( 'limit', 'The maximum number of log lines', 500 )
+	.option( 'format', 'Output the log lines in CSV or JSON format', 'console' )
```

Comment:
Subjective, but I might suggest `plain` as the default format name. All formats get sent to the console. Naming the format `console` might imply that the other formats are written to disk.

Reply:
`text` makes sense 👍

Reply:
8ec6b6b